src/Modal.jsx

```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
     return null;
   }
 
-  render = () => (
-    <div>
+  render = () => {
+    
+    const {
+      wrapperProps,
+      className,
+      visible,
+      onClickBackdrop,
+      onOK,
+      onCancel,
```

Comment:
`onOK` and `onCancel` only apply to the `ConfirmModal` component, so they shouldn't be excluded here.  Perhaps a similar thing in `ConfirmModal` to do that would be good though.